packages/compiler/core/program.ts

```diff
@@ -97,6 +97,13 @@ export async function createProgram(
     await loadStandardLibrary(program);
   }
 
+  // Load additional imports prior to compilation
+  if (options.additionalImports) {
+    const importScript = options.additionalImports.map((i) => `import "${i}";`).join("\n");
```

Comment:
It be great to find a more reliable way of doing imports calling in loadImports directly but this might require too much change. Might be good to have a MQ issue for that. Feel like this might get us weird error from a user input

Reply:
Yeah, this is definitely the cheap way to do it.  It seemed like you were going for a more robust solution in your PR, so perhaps this code could be ported over to that once yours is in?

Reply:
well I think we would still keep that but I guess I can do the reorganization of imports as I'm touching that code anyway

Reply:
Sounds good!